keyboards/hub16/keymaps/via/rules.mk

```diff
@@ -0,0 +1,5 @@
+VIA_ENABLE = yes
+LDO_ENABLE = yes
```

Comment:
I think you mean
```suggestion
LTO_ENABLE = yes
```
?

Reply:
Good catch, have removed LTO as it appears that the extra space is not required. Thanks!